convention-plugins/src/main/kotlin/conventions-annotations-processor.gradle.kts

```diff
@@ -19,7 +33,7 @@ afterEvaluate {
     if (extension.useKoin) {
         ksp {
             // enable compile time check
-            arg("KOIN_CONFIG_CHECK", "true")
+            arg("KOIN_CONFIG_CHECK", "false")
```

Comment:
I'm disabling this for now because Koin annotations somehow refuses to detect dependencies with `named` qualifier from the same subproject.